<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskForm</class>
 <widget class="QWidget" name="TaskForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Task</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Logo.qrc">
    <normaloff>:/logo/WindowLogo.png</normaloff>:/logo/WindowLogo.png</iconset>
  </property>
  <widget class="QLabel" name="TaskNumber">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>361</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Задание номер</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="AnswerLine">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="SendAnswerButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>250</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Ответить</string>
   </property>
  </widget>
  <widget class="QLabel" name="ResultCheckedAnswer">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>250</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Верно!</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ErrorLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ответ не может быть пустым!</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Logo.qrc"/>
 </resources>
 <connections/>
</ui>
